internal/page/certificateprovider/confirm_dont_want_to_be_certificate_provider.go

```diff
@@ -0,0 +1,70 @@
+package certificateprovider
```

Comment:
I'm not sure how I feel about being able to access this page with just an LPA ID (via the red button on provide certificate). I considered adding in the session store to get the login sessions but the logic was getting a bit messy but it feels like we need something here to stop anyone with the ID being able to get here and sever a CP.

Reply:
I think having 2 pages might make this all make more sense. You have one which is normal for someone logged in, and another which does the check for the reference number. Then we'd at least have that for safety.

Reply:
Also you could make it so `/certificate-provider-enter-reference-number-opt-out` stores the LpaID in the session cookie, so when you get to the next page you aren't relying on something that can be manually changed. We know you must have entered a ref number for that LpaID to be there.

Reply:
Yeah makes sense, I'll split this out.